# MPC parameters
state_costs:
  x: 1
  dx: 0.1
  z: 5
  dz: 0.5
  att: 4
  datt: 4
  droll: 5

input_costs:
  servo: 5
  thrust: 0.01
  torque: 0.01

# constraints (+ used for scaling)
min_z: -1
min_dz: -0.6

max_dx: 1.2
max_dz: 2
max_datt: 0.6
max_attitude_angle: 90

# scaling
scaling_x: 2
scaling_z: 4


# speed/performance
horizon_length: 1.0

max_sqp_iter: 1
max_qp_iter: 25
max_line_search_iter: 4
mpc_period: 0.0333
feedforward_period: 0.0333

weight_scaling: 64